src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -304,6 +363,37 @@
  * If you create both a {@code MessageFormat} and {@code ChoiceFormat}
  * programmatically (instead of using the string patterns), then be careful not to
  * produce a format that recurses on itself, which will cause an infinite loop.
+ *
+ * <h2>Formatting Date and Time</h2>
+ *
+ * MessageFormat provides patterns that support the date/time formatters in the
+ * {@link java.time.format} and {@link java.text} packages. Consider the following three examples,
+ * with a date of 11/16/2023:
+ *
+ * <p>1) a <i>date</i> {@code FormatType} with a <i>full</i> {@code FormatStyle},
+ * {@snippet lang=java :
+ * Calendar cal = Calendar.getInstance();
+ * cal.set(123 + 1900, 10, 16);
+ * Object[] arg = {cal.getTime()};
```

Comment:
Maybe with the following, better aligned with `java.time`'s example?
```
Object[] arg = {new GregorianCalendar(2023, Calendar.NOVEMBER, 16).getTime()};
```

Reply:
Thanks, that's a lot more concise, fixed.